src/github/commit/index.ts

```diff
@@ -32,7 +32,10 @@ interface IGitHubCommitArgs {
  */
 export const execute = async (args: IGitHubCommitArgs = {}): Promise<string | undefined> => {
   let retVal: string | undefined =
-    process.env.CIRCLE_SHA1 || process.env.TRAVIS_PULL_REQUEST_SHA || process.env.TRAVIS_COMMIT;
+    process.env.CIRCLE_SHA1 ||
+    process.env.TRAVIS_PULL_REQUEST_SHA ||
+    process.env.TRAVIS_COMMIT ||
+    process.env.GITHUB_SHA;
```

Comment:
[super nit] I'd move the GITHUB stuff to top priority evaluation – represents our FIFO journey from Travis -> CircleCI -> GH Actions :wink: